test/TestPercentageMath.sol

```diff
@@ -24,6 +32,16 @@ contract PercentageMathFunctionsRef {
     function percentDiv(uint256 x, uint256 y) public pure returns (uint256) {
         return PercentageMathRef.percentDiv(x, y);
     }
+
+    function percentAvg(
+        uint256 x,
+        uint256 y,
+        uint256 percentage
+    ) public pure returns (uint256) {
+        return
+            PercentageMathRef.percentMul(x, PercentageMathRef.PERCENTAGE_FACTOR - percentage) +
+            PercentageMathRef.percentMul(y, percentage);
+    }
```

Comment:
Use the same no? As it will be the reference after

Reply:
This is what's done here, except I did not copy the revert